pkg/media/audiotrack.go

```diff
@@ -257,10 +264,12 @@ func NewPCMRemoteTrack(track *webrtc.TrackRemote, writer *media.WriteCloser[medi
 	// resampledPCMWriter resamples the PCM16 samples from DefaultOpusSampleRate to targetSampleRate and
 	// writes them to the writer. If no resampling is needed, we directly point resampledPCMWriter to writer.
 	var isResampled bool
-	resampledPCMWriter := *writer
+	var resampledPCMWriter media.WriteCloser[media.PCM16Sample]
 	if targetSampleRate != DefaultOpusSampleRate {
-		resampledPCMWriter = media.ResampleWriter(*writer, targetSampleRate)
+		resampledPCMWriter = media.ResampleWriter(writer, DefaultOpusSampleRate)
```

Comment:
Not for now.

For later, should this be gotten from RemoteTrack actual value?

Reply:
Yeah, currently even SIP is assuming it to be 48k always

Reply:
Yeah, will leave it at 48k sample rate for now.